src/Core/NotificationCenter/Queries/Interfaces/IGetNotificationStatusForUserQuery.cs

```diff
@@ -0,0 +1,9 @@
+﻿#nullable enable
+using Bit.Core.NotificationCenter.Entities;
+
+namespace Bit.Core.NotificationCenter.Queries.Interfaces;
+
+public interface IGetNotificationStatusForUserQuery
+{
+    Task<NotificationStatus> GetByNotificationIdAndUserIdAsync(Guid notificationId);
```

Comment:
I have left this as a remainder from first PR draft, but let me know if you think it's not useful and should delete it instead. @withinfocus

Reply:
To be fair, none of the Queries returns the statuses. Maybe this should be a combination of `IGetNotificationsForUserQuery` and `IGetNotificationStatusForUserQuery`, with modification to be able to get multiple`NotificationStatus`
```suggestion
    Task<IEnumerable<NotificationStatus>> GetByNotificationIdAndUserIdAsync(IEnumerable<Guid> notificationId);
```

Reply:
That's a valid area of potential improvement, but should we just make a custom view that blends notifications with their status to streamline it as one query to the database? Your suggestion is cleaner code and showcasing of the model, but we also want to be performant.